doc/release/1.8.0-notes.rst

```diff
@@ -26,104 +26,876 @@ For running on PyPy, PyPy3 6.0+ is required.
 Highlights of this release
 **************************
 
+- A sparse array API has been added for early testing and feedback; users
+  may expect backwards compatibility changes in future releases
```

Comment:
@tylerjereddy I wasn't quite sure what this meant "may expect backwards compatibility changes".

Reply:
Could change pretty substantially in the future is what it means--it was a suggestion from @ev-br above and I liked it. It sounds like you and Ralf don't like it... anyway, you can always propose a fix and come to an agreement with Ralf I guess...